examples/lambda-in-vpc.tf

```diff
@@ -0,0 +1,19 @@
+module "spectral_lambda_integration" {
+  source = "github.com/SpectralOps/spectral-terraform-lambda-integration"
+
+  integration_type = "gitlab"
+
+  env_vars = {
+    # Required environment variables 
+    SPECTRAL_DSN          = "MySpectralDSN"
```

Comment:
I think better avoid spectral dsn in the examples - the best practice is to use vault since this is a secret

Reply:
Fixed:
https://github.com/SpectralOps/spectral-terraform-lambda-integration/pull/20/commits/9007bf3ed2f04aa25b776a9c82f2d4bfb7fc9fb7